src/procedures/procedures.11tydata.yaml

```diff
@@ -362,14 +390,14 @@ procedureReassembleCase:
           <li>Make sure the USB connectors are facing upwards, away from you.</li>
         </ul>
     - title: Reassembling - top part
-      image: reassemble-step-3-insert-pcb-and-bottom-plate.webp
+      image: close_cover_480.webp
       content: |
         <ul>
           <li>Turn the enclosure upside down.</li>
           <li>Then, slide the push buttons into the openings of the case.</li>
         </ul>
     - title: Mounting the case
-      image: cm5_insert-thumb-screws.jpeg
+      image: cm5_insert-thumb-screws.jpg
```

Comment:
Hm, I see that sneaked in with #264.

How about renaming the files instead? Seems we use jpeg everywhere else.